torch/__init__.py

```diff
@@ -84,20 +84,20 @@
     for dll in dlls:
         is_loaded = False
         if with_load_library_flags:
-            res = kernel32.LoadLibraryExW(dll, 0, 0x00001100)
+            res = kernel32.LoadLibraryExW(dll, None, 0x00001100)
             last_error = ctypes.get_last_error()
-            if res == 0 and last_error != 126:
+            if res is None and last_error != 126:
                 err = ctypes.WinError(last_error)
                 err.strerror += ' Error loading "{}" or one of its dependencies.'.format(dll)
                 raise err
-            elif res != 0:
+            elif res:
```

Comment:
Because `0` would not be a valid return code, wouldn't it?
```suggestion
            elif res is not None:
```